src/library/itunes/itunesdao.cpp

```diff
@@ -46,22 +48,27 @@ void ITunesDAO::initialize(const QSqlDatabase& database) {
     m_applyPathMappingQuery = QSqlQuery(database);
 
     m_insertTrackQuery.prepare(
-            "INSERT INTO itunes_library (id, artist, title, album, "
-            "album_artist, genre, grouping, year, duration, "
-            "location, rating, comment, tracknumber, bpm, bitrate) "
-            "VALUES (:id, :artist, :title, :album, :album_artist, "
-            ":genre, :grouping, :year, :duration, :location, "
-            ":rating, :comment, :tracknumber, :bpm, :bitrate)");
-
-    m_insertPlaylistQuery.prepare("INSERT INTO itunes_playlists (id, name) VALUES (:id, :name)");
-
-    m_insertPlaylistTrackQuery.prepare(
-            "INSERT INTO itunes_playlist_tracks (playlist_id, track_id, "
-            "position) VALUES (:playlist_id, :track_id, :position)");
-
-    m_applyPathMappingQuery.prepare(
-            "UPDATE itunes_library SET location = replace( location, "
-            ":itunes_path, :mixxx_path )");
+            QStringLiteral("INSERT INTO %1 (id, artist, title, album, "
+                           "album_artist, genre, grouping, year, duration, "
+                           "location, rating, comment, tracknumber, bpm, bitrate) "
+                           "VALUES (:id, :artist, :title, :album, :album_artist, "
+                           ":genre, :grouping, :year, :duration, :location, "
+                           ":rating, :comment, :tracknumber, :bpm, :bitrate)")
+                    .arg(ITUNES_LIBRARY_TABLE));
```

Comment:
Even though this is not user input, please lets try to avoid string manipulations on queries whenever possible regardless. 

```suggestion
            QStringLiteral("INSERT INTO :table (id, artist, title, album, "
                           "album_artist, genre, grouping, year, duration, "
                           "location, rating, comment, tracknumber, bpm, bitrate) "
                           "VALUES (:id, :artist, :title, :album, :album_artist, "
                           ":genre, :grouping, :year, :duration, :location, "
                           ":rating, :comment, :tracknumber, :bpm, :bitrate)"));
               m_insertTrackQuery.bindValue("table", ITUNES_LIBRARY_TABLE);
```

Reply:
Looks like that's not really possible with Qt: https://stackoverflow.com/questions/15902859/qsqlquery-with-prepare-and-bindvalue-for-column-name-sqlite

Reply:
Thank you. I didn't know that. Then disregard my suggestion.